web/src/client/components/terminal-quick-keys.ts

```diff
@@ -9,6 +9,7 @@ const TERMINAL_QUICK_KEYS = [
   { key: 'CtrlExpand', label: '⌃', toggle: true, row: 1 },
   { key: 'F', label: 'F', toggle: true, row: 1 },
   { key: 'Tab', label: 'Tab', row: 1 },
+  { key: 'shift_tab', label: '⇤', row: 1 },
```

Comment:
•	⇤ = LEFTWARDS ARROW TO BAR (U+21E4), usually used to mean “reverse tab”

TIL!